Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/bean/ConceptsMergeBean.java

```diff
@@ -0,0 +1,164 @@
+package edu.asu.conceptpower.app.bean;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import edu.asu.conceptpower.app.core.POS;
+import edu.asu.conceptpower.core.ConceptType;
+
+public class ConceptsMergeBean {
```

Comment:
But these are two different things, and you can just as easily use one object for the input values for the page and another one for the submitted values. Or do you need all type ids and all list names in the submitted object to create the new merged concept?